### ISSUE TITLE
3D visualization tool crashing software

### ISSUE THREAD

**Issue by PatrinaPellett** [oxfordni/support#2]
Attempting to use the 3D visualization tool on Cersei at Penn resulted in NimOS crashing. Here is the sequence of events:

1. Load 3D localization dataset
2. Click 3D Visualization tool
3. Select region in image and press enter
4. Microscope disconnects and software closes. Goes to screen about debugging the nanoimager software.

I updated the software to the most recent version and had the same result. Screenshot of older version attached. 

With Cersei at Syracuse, the tool worked fine. At Syracuse we were using the laptop and at Penn we used the desktop, but with the same software versions. 

![20180207_150413503_ios](https://user-images.githubusercontent.com/36457627/36259856-2cbb4c32-1225-11e8-8825-6257372c1a1a.jpg)

**Comment by trygub** [comment]
We are unable to reproduce this issue. Would it be possible to have access to the data set you were using? Also, it would be helpful to know if this behaviour is observed on more than one data set / machine?

**Comment by PatrinaPellett** [comment]
Yes, I have the dataset and tried it on my laptop. It worked fine. Will send you the locb file via email.
